### DISCUSSION TITLE
turborepo with yarn v2

### DISCUSSION THREAD

**Discussion by dwalker93** [vercel/turborepo#456] [Help] [answered]
Is turborepo currently supporting yarn v2 or not? Im trying to set a project with yarn v2 but always giving yarn errors. but working fine with yarn v1.22.1.

**Comment by richardvanbergen** [comment, marked as answer]
I've had a lot of problems with every project I've tried to use yarn berry in PnP mode. It works for me if I use `nodeLinker: node-modules`

**Reply by dwalker93** [reply]
Thanks @richardvanbergen with that option turbo also working fine with yarn v3.1.1

**Reply by richardvanbergen** [reply]
Cool! Shame I haven't been able to get it to work with zero install though...

Don't forget to update your gitignore for the non zero-install configuration too! https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
